### DISCUSSION TITLE
Weather

### DISCUSSION THREAD

**Discussion by CHristapherenese** [kutu/pyirsdk#45] [General]
First of all thank you for your great work. We have developed an Admin Control and now made a connection to the simulator via your using this SDK and very happy. I am vaguely familiar with programming and have someone implementing this work. 

My question at this point is what are your plans for weather, we have developed a nice widget to show this. 

Again, thank you
![IMG_3240](https://github.com/user-attachments/assets/a96c6429-61bb-4c9a-9ca8-d6ffcb01e8d4)

**Comment by kutu** [comment]
You will have to ask iRacing.

**Reply by CHristapherenese** [reply]
Thank you for the quick response. I will do, but I thought you had something that was in and removed. Any help would be appreciated as I a not a programmer and my programmer is not an iRacer.

Additionally, is there a way to contribute to you monetarily?

**Reply by kutu** [reply]
> you had something that was in and removed

because iRacing removed it

Looking at your project, you are trying to manipulate ongoing race as an admin, you can't change weather mid-race.

You can't contribute to me monetarily.

**Comment by CHristapherenese** [comment]
Thx, sorry for the confusion, with the weather we are are not trying to manipulate just report it on a small LCD. Obviously, we can get it going with some of the information but not the forecasting. Will probably add some flag data and driver positions next to the car (driver right etc.) 

![image](https://github.com/user-attachments/assets/0bfe8c13-cfb4-4469-9bb5-4b0a64704385)

**Comment by CHristapherenese** [comment]
The picture I originally sent is the AdminBoxx 4.0 which uses the API for feed back. It is working nicely. Our previous 3.o was done in MakeCode and a bit vulnerable to the public.

https://www.adminboxx.com/#/

Not going to get rich on this, just support new leagues and the community in a whole. Also to feed my habit on iRacing.
